.golangci.yml

```diff
@@ -0,0 +1,35 @@
+version: "2"
+linters:
+  disable:
+    - errcheck      # TODO(rsned): Enable once outstanding lint bugs are fixed.
+    - mnd
+    - paralleltest
+    - staticcheck   # TODO(rsned): Enable once outstanding lint bugs are fixed.
+    - testpackage
+    - unused
+  exclusions:
+    generated: lax
+    presets:
+      - comments
+      - common-false-positives
+      - legacy
+      - std-error-handling
+    paths:
+      - third_party$
```

Comment:
What do these paths refer to?

Reply:
They were just in the examples. I assume they come from peoples common practice of naming other bits of code that would commonly be ignored.  I've dropped them.